apps/emqx_bridge_kafka/test/emqx_bridge_kafka_impl_producer_SUITE.erl

```diff
@@ -629,16 +676,28 @@ t_send_message_with_headers(Config) ->
             )
         end
     ),
-    {ok, {_, [KafkaMsg]}} = brod:fetch(kafka_hosts(), KafkaTopic, 0, Offset),
+    {ok, {_, KafkaMsgs}} = brod:fetch(kafka_hosts(), KafkaTopic, 0, Offset),
     ?assertMatch(
-        #kafka_message{
-            headers = [
-                {<<"clientid">>, _},
-                {<<"payload">>, <<"\"payload\"">>}
-            ],
-            key = BinTime
-        },
-        KafkaMsg
+        [
+            #kafka_message{
+                headers = [
+                    {<<"foo">>, <<"\"bar\"">>},
```

Comment:
Q: should those quotes in value exist?  They seem a bit odd.  :thinking: 

Shouldn't it be:

 ```suggestion
                     {<<"foo">>, <<"bar">>},
```
?

Reply:
it's because the `kafka_header_value_encode_mode` is set to json, so the string is being surrounded by quotes.